.test-infra/jenkins/job_PerformanceTests_PubsubIO_Python.groovy

```diff
@@ -41,7 +42,7 @@ def psio_test = [
                 metrics_dataset      : 'beam_performance',
                 metrics_table        : 'psio_io_2GB_msg_results',
                 input_options        : '\'{' +
-                        '"num_records": 2097152,' +
+                        '"num_records": 2097152' +
```

Comment:
Why did you remove that comma?

Reply:
It sounds stupid but my cat went through my keyboard and I deleted too many digits and overlooked it